server/prisma/generator/setupRoles.ts

```diff
@@ -20,6 +20,20 @@ const setupRoles = async (
 ): Promise<void> => {
   const usersData: Prisma.chapter_usersCreateManyInput[] = [];
   const subscribeIterator = makeBooleanIterator();
+
+  // We create one admin for chapter one, and none, so that we can test what
+  // happens if an this admin tries to make changes to a chapter that they don't
+  // have access to.
```

Comment:
```suggestion
  // We create one admin for chapter one, and none, so that we can test what
  // happens if this admin tries to make changes to a chapter that they don't
  // have access to.
```
There might be also something missing (or too much) in the first part of sentence. _and none_ sounds a bit out of place.

Reply:
Ooof, thanks.  It should have said 'and no admins for the other chapters'.  I'll clean that up after I merge in main.